### ISSUE TITLE
Idyll incorrectly formats a code block

### ISSUE THREAD

**Issue by suhr** [idyll-lang/idyll#495]
The code block (using ```):

```
                  C  g  d  a  e  b f♯
               f  C  g  d  a  e  b
            b♭ f  C  g  d  a  e
         e♭ b♭ f  C  g  d  a
      a♭ e♭ b♭ f  C  g  d
   d♭ a♭ e♭ b♭ f  C  g
g♭ d♭ a♭ e♭ b♭ f  C
```

Idyll output:

```
C  g  d  a  e  b f♯
               f  C  g  d  a  e  b
            b♭ f  C  g  d  a  e
         e♭ b♭ f  C  g  d  a
      a♭ e♭ b♭ f  C  g  d
   d♭ a♭ e♭ b♭ f  C  g
g♭ d♭ a♭ e♭ b♭ f  C
```

**Comment by mathisonian** [comment]
Thanks for reporting this @suhr. I'm working on a fix in #497, and this patch will go out in the next release. I'll close this issue once that happens.

**Comment by mathisonian** [comment]
This is fixed in the latest release, `v4.1.0`.
